### PR TITLE
Simplify concat

### PR DESCRIPTION
**Pull request by wasade**
This PR allows for (a) calling `biom.Table.concat` with a single table as well as an iterable of tables and (b) provides a helper method (`biom.concat`) that removes the need for the common pattern `table = tables[0].concat(tables[1:])` in favor of `table = biom.concat(tables)`

### CODE REVIEW

File: biom/__init__.py [hunk 1]
```diff
@@ -73,5 +73,10 @@
                        {'environment': 'A'}], input_is_dense=True)
 
 
+def concat(tables, *args, **kwargs):
+    """Wrapper for biom.Table.concat which requires a table instance"""
```
Comment (ElDeveloper) [inline biom/__init__.py:77]: Can you add the little bit of numpy doc here? Seems like a very handy function that would be good to have well documented.

File: biom/__init__.py [hunk 2]
```diff
@@ -73,5 +73,10 @@
                        {'environment': 'A'}], input_is_dense=True)
 
 
+def concat(tables, *args, **kwargs):
```
Comment (ElDeveloper) [inline biom/__init__.py:76]: Can you add a test for this function?

**Review by ElDeveloper — CHANGES_REQUESTED** [review]
A few small comments, looks good and helpful!!

File: biom/__init__.py (hunk 2 above)
Comment (wasade) [reply biom/__init__.py:76]: yup, can do!

File: biom/__init__.py (hunk 1 above)
Comment (wasade) [reply biom/__init__.py:77]: Ya sounds good

**Review by ElDeveloper — APPROVED** [review]

**Comment by wasade** [comment]
Thanks!!

**Pull request merged**